lib/bap_image/sectionRef.mli

```diff
@@ -0,0 +1,38 @@
+(** Signle section in the list of sections int the object file *)
```

Comment:
Ok, then just convert this string to `Invalid_argument` exception. `Or_error.t` is for handling errors in libraries. And believe me, there is no such function like `caml_create_or_error`. Actually, it is impossible to create value of type `Or_error.t` in C, until you really write a separate library that will expose this interface to C.